docs/InfiniteList.md

```diff
@@ -267,4 +296,45 @@ const ProductList = () => {
 };
 ```
 
-`useInfiniteListController` returns callbacks to sort, filter, and paginate the list, so you can build a complete List page.
\ No newline at end of file
+`useInfiniteListController` returns callbacks to sort, filter, and paginate the list, so you can build a complete List page.
+
+## Access Control
```

Comment:
I rephrased the doc for this component, and I think you should do the same for the others